tensorboard/webapp/experiments/types.ts

```diff
@@ -22,6 +22,7 @@ export declare interface Experiment {
   hparams?: string;
   tags?: string[];
   related_links?: Array<{name: string; url: string}>;
+  state?: string;
```

Comment:
It's not a particularly useful or insightful name. But I see it comes from https://google.aip.dev/164 and https://google.aip.dev/216.

Can you perhaps type it like `state?: 'active'|'hidden'|'whateverelse'` and include a link to the AIP and explain that the state values we've chosen are specific to Google use cases?

Reply:
Done.